src/screens/consul/LoginRegister/ConsulRegisterScreen.js

```diff
@@ -152,11 +152,11 @@ export const ConsulRegisterScreen = ({ navigation }) => {
                 rightIcon={rightIcon(secureTextEntry, setSecureTextEntry)}
                 validate
                 rules={{
-                  required: true,
+                  required: texts.consul.passwordError,
                   minLength: { value: 8, message: texts.consul.passwordLengthError },
                   validate: (value) => value === pwd || texts.consul.passwordDoNotMatch
                 }}
-                errorMessage={errors['password-repeat'] && `${texts.consul.passwordError}`}
+                errorMessage={errors['password-repeat'] && `${errors['password-repeat'].message}`}
```

Comment:
why are the values after `&&` always interpolated everywhere in all the changed files? i guess they do not need to be strings?

Reply:
Yes that's true. Actually, only string returns here. But I still wanted to put it in string just to be sure.

Reply:
just to be sure for what?

Reply:
eddf2ad3a7cb0b2c717a22027b3979abd3e0388c